cmd/cosign/cli/verify/verify_blob.go

```diff
@@ -88,14 +68,20 @@ type VerifyBlobCmd struct {
 // nolint
 func (c *VerifyBlobCmd) Exec(ctx context.Context, blobRef string) error {
 	var cert *x509.Certificate
-	var bundle *bundle.RekorBundle
+	var chain []*x509.Certificate
+	opts := make([]static.Option, 0)
 
 	// Require a certificate/key OR a local bundle file that has the cert.
+	if options.NOf(c.KeyRef, c.CertRef, c.Sk, c.BundlePath) == 0 {
+		return fmt.Errorf("please provide a cert to verify against via --certificate or a bundle via --bundle")
+	}
+
+	// We can't have both a key and a security key
```

Comment:
What about a key/sk and a cert?

Reply:
Agree

Reply:
key, sk, and cert are mutually exclusive. Changed to that condition.